README.md

```diff
@@ -75,10 +75,9 @@ The action can be configured with the following arguments:
   this update
 
 - `edit-pr-comment` - (optional) Edit previous PR comment instead of posting new
-  one.  
-  **PLEASE NOTE:** that as of 3.2.0 of the Action, this now defaults to `true`.
-  This is in an effort to reduce verbosity - if you want to have a comment per
-  PR run, please ensure that you set this to `false`.
+  one. **PLEASE NOTE:** that as of 3.2.0 of the Action, this now defaults to
+  `true`. This is in an effort to reduce verbosity - if you want to have a
+  comment per PR run, please ensure that you set this to `false`.
```

Comment:
Ah, these changes seem rather unrelated to the PR at hand. AFAIK they shouldn't impact markdown rendering at all. Would you mind backing them out?